storage_v2/depstore/reader.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package depstore
+
+import (
+	"context"
+	"time"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+// Reader can load service dependencies from storage.
+type Reader interface {
+	GetDependencies(ctx context.Context, endTs time.Time, lookback time.Duration) ([]model.DependencyLink, error)
```

Comment:
1. maybe we should have a query params struct instead of explicit params, for future extensibility
2. the trace store TraceQueryParameters is defining the time range with two timestamps, maybe we should use that here too instead of less intuitive endTs/lookback

Reply:
@yurishkuro done